include/Base.h

```diff
@@ -0,0 +1,26 @@
+#ifndef BASE_H
+#define BASE_H
+#include <vector>
+
+
+class BaseOperations
+{
+	public:
+	int ID=100;
```

Comment:
What is 100 ? should be defined as const expressiong

Reply:
Initialization of ID,ID starts with 100.